### PR TITLE
Closes #79:

### LINKED ISSUE #79: Check backend used credentials
Check that the WS module uses the client provided credentials with the Alignak backend.

1/ the client credentials are used to allow WS access

2/ the client credentials are used to interfere with the backend

Anyway, what is the best behavior to implement ?

### PR DESCRIPTION
**Pull request by mohierf**
- check that the backend is used with the WS client credentials
- set test user as non admin with correct CRUD

### CODE REVIEW

File: alignak_module_ws/ws.py [hunk 1]
```diff
@@ -145,7 +146,7 @@ def __init__(self, mod_conf):
             # and the backend is yet connected and authenticated
             self.backend.token = getattr(mod_conf, 'token', '')
             self.backend.authenticated = (self.backend.token != '')
-            self.backend_available = False
+            # self.backend_available = False
```
Comment (ddurieux) [inline alignak_module_ws/ws.py]: Remove instead comment ;)

File: alignak_module_ws/ws.py [hunk 2]
```diff
@@ -515,13 +516,13 @@ def getHost(self, host_name):
...
         ws_result = {'_status': 'OK', '_result': [], '_issues': []}
         try:
-            if not self.backend_available:
-                self.backend_available = self.getBackendAvailability()
-            if not self.backend_available:
-                ws_result['_status'] = 'ERR'
-                ws_result['_issues'].append("Alignak backend is not available currently. "
-                                            "Host information cannot be fetched.")
-                return ws_result
+            # if not self.backend_available:
```
Comment (ddurieux) [inline alignak_module_ws/ws.py]: Same here, delete instead comment

**Review by ddurieux — CHANGES_REQUESTED** [review]

File: alignak_module_ws/ws.py (hunk 1 above)
Comment (mohierf) [reply alignak_module_ws/ws.py]: Ok

**Comment by mohierf** [comment]
I made the required modifications and I will merge this PR

**Comment by ddurieux** [comment]
It will be better to squash before merge, but it's too late :(

**Comment by mohierf** [comment]
Too late :( This is because this branch was rebased on develop ... quite strange indeed!

**Pull request merged**